queries/jsx/textobjects.scm

```diff
@@ -0,0 +1,6 @@
+; inherits: ecma
```

Comment:
i don't think this include is necessary. jsx is just a optional mixin for js and tsx

Reply:
The reason I added this was because I am not sure if a JSX file will be resolved to both jsx and JavaScript or just, jsx. If it is just jsx you probably want your JavaScript queries too right? I included ecma because JavaScript would result in circular dependency